src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -783,6 +781,36 @@ internal bool IsAutoProperty
                 return null;
             }
         }
+
+        private bool AllowInitializer
+        {
+            get
+            {
+                // PROTOTYPE(semi-auto-props): Fix implementation for semi auto properties.
+                return (_setMethod is null && _getMethod?.BodyShouldBeSynthesized == true) ||
+                    _setMethod?.BodyShouldBeSynthesized == true;
```

Comment:
@AlekseyTs The test failure is due to a difference in the error scenario where there are no accessors at all. I think it doesn't matter whether we produce both `ErrorCode.ERR_PropertyWithNoAccessors` and `ErrorCode.ERR_InitializerOnNonAutoProperty` or only `ErrorCode.ERR_PropertyWithNoAccessors`.

But I can special case the no accessors case if you'd like.

Reply:
> But I can special case the no accessors case if you'd like.

Let's update the baseline for now.